docs/code-quality/ca1010.md

```diff
@@ -28,7 +28,7 @@ ms.workload:
 
 A type implements the <xref:System.Collections.IEnumerable?displayProperty=fullName> interface but does not implement the <xref:System.Collections.Generic.IEnumerable%601?displayProperty=fullName> interface, and the containing assembly targets .NET. This rule ignores types that implement <xref:System.Collections.IDictionary?displayProperty=fullName>.
 
-By default, this rule only looks at externally visible types, but this is [configurable](#configurability).
+By default, this rule only looks at externally visible types, but this is [configurable](#configurability). You can also configure additional interfaces requiring there generic pendent to be implemented.
```

Comment:
```suggestion
By default, this rule only looks at externally visible types, but this is [configurable](#configurability). You can also configure additional interfaces requiring their generic equivalent to be implemented.
```

Reply:
After reading the new content, trying to clarify phrasing a bit.

Reply:
Let me know if it seems accurate. Not sure that it covers this example: ISomething->System.Collections.Generic.IEnumerable`\``1,
which doesn't seem like the generic equivalent.
Maybe this is better and simpler:
"You can also configure additional interfaces to require that a generic interface be implemented."

Reply:
> "You can also configure additional interfaces to require that a generic interface be implemented."

LGTM